rust/iso-8601/src/lib.rs

```diff
@@ -0,0 +1,296 @@
+use chrono::{DateTime, FixedOffset, TimeZone};
+
+pub fn parse_datetime(str: &str) -> Option<DateTime<FixedOffset>> {
```

Comment:
In rust it libraries especially it is not common to put functions in struct impls. 

So the uses of your libraries are concise and do not include a lot of random functions.